compiler/rustc_infer/src/infer/error_reporting/nice_region_error/different_lifetimes.rs

```diff
@@ -21,7 +21,7 @@ impl<'a, 'tcx> NiceRegionError<'a, 'tcx> {
     ///
     /// Consider a case where we have
     ///
-    /// ```compile_fail,E0623
+    /// ```compile_fail
```

Comment:
This test doesn't provide `E0623`, that's why it's removed. Can't see the error code of the error. It just raises `error: lifetime may not live long enough`

Reply:
Not all errors have codes, that's fine.